frontend/src/static/css/v_zoom.scss

```diff
@@ -1,15 +1,21 @@
 @import 'colors';
 
+.panel-title {
+  @include large-font;
+}
+
 .zoom {
   &__title {
     &--granularity {
+      @include mini-font;
       margin-top: .5rem;
     }
 
     &--tags {
       margin: .25rem 0 .25rem 0;
 
       .tag {
+        @include mini-font;
```

Comment:
isn't this unnecessary with the one at line 32?

Reply:
One is within `__title--tags.tag` whereas the other is within `__title.tag` so I think might be different?

Reply:
@ang-zeyu Do you think it's same?

Reply:
yup, for the current usages ( only two, one under `zoom__title` and another `zoom__day` ) they would be the same;
the `&--tags` version only overrides the margin

Reply:
Sure